bridge-sdk/connectors/nep141-connector/src/nep141_connector.rs

```diff
@@ -282,6 +319,8 @@ impl Nep141Connector {
         )
         .await?;
 
+        tracing::info!(tx_hash = format!("{:?}", tx_hash), "Sent finalize withdraw transaction");
```

Comment:
I also want to remove word "message" from all logs, but I am not sure that it is possible to do in this lib.

Reply:
Removed the message label